packages/cli/create-strapi-app/tsconfig.eslint.json

```diff
@@ -1,3 +1,5 @@
 {
-  "extends": "./tsconfig.json"
+  "extends": "./tsconfig.json",
+  "include": ["src", "packup.config.ts", "package.json"],
+  "exclude": ["node_modules", "src/resources/**/*"]
```

Comment:
What's in src/resources?

Reply:
Shouldn't this be templates if you wanted to exclude them?

Reply:
Ha missed that one I have to remove resources it was the previous files